lib/components/Slider/Slider.tsx

```diff
@@ -88,86 +87,83 @@ export const Slider = ({ label, onValueChange, value, rightLabel, disabled }: Sl
 
   return (
     <div
-      className={cx(
-        'ds-flex ds-h-[40px] ds-rounded-[20px] ds-bg-white ds-justify-between ds:min-w-full sm:ds-min-w-[414px]',
```

Comment:
I assumed `ds:min-w-full` was a typo so I replaced it with `ds-min-w-full`

Reply:
Good find!